boltdesignsystem/bolt
# DS-328 fix starter kit CSS order and misc visual bugs

## Jira

https://pegadigitalit.atlassian.net/browse/DS-328

## Summary

1. Fixes a bug where the form select input's text overlaps the chevron down icon
2. Fixes a bug where the underline on hover is not displaying for the "reversed underline" link variant
3. Fixes a bug where the page header's CTA text is off centered in mobile view
4. Fixes a bug where the Elements CSS is loaded after Components CSS

## Details

1. Added proper padding to the select input
2. Added missing text-decoration hover styles to link component and text link element
3. Re-ordered the elements and components within `boltrc.js` in starter kit (which also fixes the page header's CTA text alignment)
4. Removed layouts-list from `package.json` and `boltrc.js`, because it was causing the List component to break.

## How to test

Run the branch locally and check form, link, and page header docs. Make sure they are displaying properly now.

## Release Notes

**Button Group**
Removed `contentItems` prop. Use `buttons` instead.
